packages/ui/react-ui-editor/src/components/TextEditor/TextEditor.tsx

```diff
@@ -18,13 +18,14 @@ import React, {
   useMemo,
 } from 'react';
 
+import { DocAccessor } from '@dxos/echo-schema';
 import { log } from '@dxos/log';
 import { useThemeContext } from '@dxos/react-ui';
 import { focusRing } from '@dxos/react-ui-theme';
 import { isNotFalsy } from '@dxos/util';
 
-import { createBasicBundle, createMarkdownExtensions, editorMode } from '../../extensions';
-import { type EditorModel } from '../../hooks';
+import { createMarkdownExtensions, editorMode } from '../../extensions';
+import { createBasicExtensions, createThemeExtensions, type EditorModel } from '../../hooks';
```

Comment:
seems weird that this is imported from hooks

Reply:
right. i'll move them once everything settles and i remove the components.